infotv/frontend/package.json

```diff
@@ -4,49 +4,46 @@
     "description": "",
     "main": "index.js",
     "scripts": {
-        "build": "webpack --verbose --progress",
-        "watch": "webpack --verbose --progress --watch",
-        "release": "webpack -p",
-        "lint": "eslint src webpack.config.js --ext .js,.jsx",
+        "build": "webpack --progress",
```

Comment:
What is this third webpack build script in addition to `watch` and `release`? Usually I'd expect the `build` script to create a production build.